homeassistant/components/media_player/kodi.py

```diff
@@ -392,7 +393,7 @@ def state(self):
         if not self._players:
             return STATE_IDLE
 
-        if self._properties['speed'] == 0 and not self._properties['live']:
```

Comment:
Why was this removed?

Reply:
see related issue from description

Reply:
:+1: Seems reasonable enough.

Going to ping @lwis since they added it originally: https://github.com/home-assistant/home-assistant/pull/4388

Reply:
The problem with this is that the state will change if the stream buffers, so if you're watching something which keeps buffering it'll trigger whatever state change actions you have set (which for me is lights, which quickly becomes annoying)